tests/test_eccv_conference.py

```diff
@@ -248,7 +248,8 @@ def conference(self, client):
             withdrawn_submission_reveal_authors=False,
             email_pcs_on_withdraw=True,
             desk_rejected_submission_public=False,
-            desk_rejected_submission_reveal_authors=False)
+            desk_rejected_submission_reveal_authors=False,
+            bidding_enabled=True)
```

Comment:
is there a test to verify when bidding_enabled is False the readers are PaperX groups?